src/Support/Field.php

```diff
@@ -22,6 +24,9 @@ abstract class Field
 {
     protected $attributes = [];
 
+    /** @var string[] */
```

Comment:
Please also add `@phpstan-var class-string[]`, thanks!

Reply:
I don't understand what this means, are you suggesting something like this?:

```suggestion
    /** @var class-string<Middleware> */
```

Reply:
Not replacing, but in addition ("also add"):
```php
@phpstan-var class-string[]
```
Actually I wouldn't add `Middleware` as a class, but technically we're doing duck-typing here => no one enforces that this has to be extending the abstract class we provide.

Reply:
I see. So like this? I see no examples in the project of any annotation like this

```suggestion
    /** 
     * @var string[]
     * @phpstan-var class-string[]
     */
```

Reply:
Exactly!